app/models/user.rb

```diff
@@ -1,13 +1,18 @@
 # User model class
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
Please enable the `:confirmable` attribute as user information needs to be confirmed through email.

<img width="704" alt="Screen Shot 2022-08-31 at 8 07 07" src="https://user-images.githubusercontent.com/82426682/187685518-94b2f53c-0832-4012-a481-0b93dc9a9d56.png">